worker/docker_client.py

```diff
@@ -40,10 +40,15 @@ class DockerClient(object):
     """
     Methods for talking to Docker.
     """
+    LIBCUDA_DIR = '/usr/lib/x86_64-linux-gnu/'
+
     def __init__(self):
         self._docker_host = os.environ.get('DOCKER_HOST') or None
+        self._nvidia_docker_host = os.environ.get('NVIDIA_DOCKER_HOST', 'localhost:3476')
```

Comment:
Because the nvidia-docker-plugin server address has a default (https://github.com/NVIDIA/nvidia-docker/wiki/nvidia-docker-plugin) but the docker daemon address doesn't have a default and is often different across hosts